bitnami/grafana-mimir/Chart.yaml

```
@@ -1,7 +1,6 @@
 annotations:
   category: Infrastructure
-  licenses: |
-    - Apache-2.0
+  licenses: "- Apache-2.0\n"
```

To maintain cohersion with the rest of the charts:

```suggestion
  licenses: Apache-2.0
```